flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -1353,6 +1353,38 @@ class TableEnvironmentTest {
       tableResult3.collect())
   }
 
+  @Test
+  def testRemoveSingleAggregateRule(): Unit = {
```

Comment:
Could you find another place to add the test? Adding this to `TableEnvironmentTest` doesn't sound right to me.

Reply:
I have moved the test to a dedicated class `RemoveSingleAggregateRuleTest`. Hopefully that makes things clearer.